recipes/dtb-mode

```diff
@@ -0,0 +1,3 @@
+(dtb-mode :repo "schspa/dtb-mode"
+          :fetcher github
+          :branch "main")
```

Comment:
If this is your default branch, you needn't specify. :)
```suggestion
          :fetcher github)
```

Reply:
this branch setup has been removed.